haskell-org-committee.markdown

```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Discourse][discourse]
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
+
+As a whole it seeks to service the open source Haskell community.
```

Comment:
Seems like we can just drop this sentence—it overlaps with the previous paragraph and doesn't add any real information.

Reply:
OK, I will update that.

Reply:
```suggestion
```